docs/guides/how-to-create.md

```diff
@@ -1,52 +1,59 @@
-# How to Create a Capability: Overview
+# How to integrate new API: Overview
 
-This guide walks you through the process of authoring new capabilities, from setting up the environment to running the capability in production.
+This guide walks you through the process of creating new API integration. You will learn how to setup the development environment, author the integration in Comlink language, integrate the API in your application and run it in production.
 
-## Understand the Superface Terms
+:::info
+
+Before diving into the process of authoring the integration in Comlink language you should check [Superface Catalog](https://superface.ai/catalog) and search through existing use cases. If your use case already exists in [Superface Catalog](https://superface.ai/catalog) follow the [Getting started](/getting-started) guide.
+
+:::
 
-First let's cover the terms you will deal with throughout this guide. The Superface capability consists of a _Profile_, _Use-Cases_, _Map_, and _Provider_. In other words:
+Superface created new domain-specific language _Comlink_ for abstracting API integrations. _Comlink_ language is designed to help developers create reusable platform agnostic API integrations.
+
+_Comlink_ language documents the integration for humans and describes the integration by structured metadata which is used by machines to perform the API calls and understand API responses.
+
+## Understand the Superface Terms
 
-> _Capability_ is a _profile_ with _use-cases_ _mapped_ to a _provider_.
+First let's cover the terms you will deal with throughout this guide. The _Comlink_ language key concepts are _Profile_, _Use Case_, _Map_, and _Provider_.
```

Comment:
I think this part should explain the stuff in very short and link to glossary instead.

Reply:
I agree. I added links to glossary and shorten the paragraph.